pkgs/sdk_triage_bot/lib/src/gemini.dart

```diff
@@ -43,7 +44,7 @@ class GeminiService {
   /// On failures, this will throw a `GenerativeAIException`.
   Future<List<String>> classify(String prompt) async {
     final result = await _query(_classifyModel, prompt);
-    final labels = result.split(',').map((l) => l.trim()).toList();
+    final labels = result.split(',').map((l) => l.trim()).toList()..sort();
```

Comment:
I personally like [`sorted`](https://api.flutter.dev/flutter/package-collection_collection/IterableExtension/sorted.html) more - but it's the same under the hood.